docs/bokeh/source/docs/dev_guide/setup.rst

```diff
@@ -664,6 +663,92 @@ setting up a development environment:
     and reinstall your development environment, following the steps in this guide
     from :ref:`the beginning <contributor_guide_setup_preliminaries>`.
 
+.. dropdown:: Slow network connections when cloning
+
+    If you are experiencing slow network connections or timeouts when attempting to clone our repository, consider performing a **shallow clone**.
+    This method downloads fewer commits, which speeds up the cloning process and reduces the amount of data transferred.
```

Comment:
I'd suggest adding some line breaks. We don't' have a hard and fast rule for ReST files but generally we try to keep narrative text line lengths to around 80-90 characters.

Reply:
OK. Will do.